pydantic/fields.py

```diff
@@ -226,6 +232,8 @@ def Field(
     lt: float = None,
     le: float = None,
     multiple_of: float = None,
+    allow_nan: bool = None,
+    allow_inf: bool = None,
```

Comment:
My only question is: do we need both of these options?

I can't imagine a scenario where we want to allow `inf` but not `nan` or visa. versa.???

I would say we should have one `allow_nan_inf` argument for clarity.

@tiangolo what do you think?